core/src/test/java/org/apache/iceberg/view/TestViewHistoryEntryParser.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.view;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TestViewHistoryEntryParser {
+
+  @Test
```

Comment:
this needs are few more tests around null and empty json and such. Some examples can be found in [`TestCounterResultParser`](https://github.com/apache/iceberg/blob/master/core/src/test/java/org/apache/iceberg/metrics/TestCounterResultParser.java#L28-L48)

Reply:
Good point, I've added more tests around null,empty, missing fields